src/algokit/core/atomic_write.py

```diff
@@ -12,10 +13,8 @@ def atomic_write(file_contents: str, target_file_path: Path, mode: Literal["a",
     temp_file_path = target_file_path.with_suffix(f"{target_file_path.suffix}.algokit~")
     try:
         # preserve file metadata if it already exists
-        try:
+        with contextlib.suppress(FileNotFoundError):
```

Comment:
Cool!

Reply:
Thanks ruff! It suggested this and I didn't even know it existed lol.